### PR TITLE
Update Anti_Cheat.lua

### PR DESCRIPTION
**Pull request by NoobBucket**
Fixes the horrendous client anti-cheat check log lag spike

### CODE REVIEW

File: MainModule/Client/Plugins/Anti_Cheat.lua [hunk 1]
```diff
@@ -698,9 +699,12 @@ return function(Vargs)
 					Detected("kick", "Bypass detected 0x48248")
 				else
 					for _, v in ipairs(Logs) do
-						if check(v.message) then
-							Detected("crash", "Exploit detected; "..v.message)
-						end
+						task.spawn(function()
+							if check(v.message) then
+								Detected("crash", "Exploit detected; "..v.message)
```
[the remark this answers is not in the document]
Comment (watameln) [reply MainModule/Client/Plugins/Anti_Cheat.lua:704]: bro i pressed it 4 times

**Comment by LiveFireExercise** [comment]
watameln 🐈🐈🐈🐈🐈🐈🐈

**Comment by Sceleratis** [comment]
Has this been tested? I thought task.spawn was unyielding (in the parent thread)

**Comment by LiveFireExercise** [comment]
yield yourself… NOW!

**Pull request closed without merging**